subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/platforms.adoc

```diff
@@ -1,6 +1,357 @@
+A bundle is basically an alias for several dependencies.
+For example, instead of declaring 3 individual dependencies like above, you could write:
+
+.Using a dependency bundle
+====
+include::sample[dir="snippets/dependencyManagement/catalogs-settings/groovy",files="build.gradle[tags=use_dependency_bundle]"]
+include::sample[dir="snippets/dependencyManagement/catalogs-settings/kotlin",files="build.gradle.kts[tags=use_dependency_bundle]"]
+====
+
+The bundle named `groovy` needs to be declared in the catalog:
+
+.Declaring a dependency bundle
+====
+include::sample[dir="snippets/dependencyManagement/catalogs-settings/groovy",files="settings.gradle[tags=catalog_with_bundle]"]
+include::sample[dir="snippets/dependencyManagement/catalogs-settings/kotlin",files="settings.gradle.kts[tags=catalog_with_bundle]"]
+====
+
+The semantics are again strictly equivalent: adding a single bundle is equivalent to adding individually the dependencies which are part of the bundle.
+
+[[sub:conventional-dependencies-toml]]
+=== The dependencies.toml file
+
+In addition to the settings API above, Gradle offers a conventional file to declare a catalog.
+If a `dependencies.toml` file is found in the `gradle` subdirectory of the root build, then a catalog will be automatically declared with the contents of this file.
+Additionally, this file can be used to declare the default plugin versions for plugins applied in the build.
+
+[WARNING]
+====
+Declaring a `dependencies.toml` file doesn't make it the single source of truth for dependencies: it's a conventional location where dependencies can be declared.
+As soon as you start using catalogs it's strongly recommended to declare all your dependencies in a catalog and not hardcode group/artifact/version strings in build scripts.
+But be aware that it's likely that some plugins add dependencies transparently, therefore dependencies defined outside of this file.
+
+Just like `src/main/java` is a convention to find the Java sources which doesn't prevent additional source directories to be declared (either in a build script or a plugin), the presence of the `dependencies.toml` file doesn't mean that it's impossible to declare dependencies elsewhere.
+
+The presence of this file does, however, suggest that most dependencies, if not all, will be declared in this file.
+Therefore, updating a dependency version, for most users, should only consists of changing a line in this file.
+====
+
+By default, the `dependencies.toml` file will be an input to the `libs` catalog.
+It is possible to change the name of the default catalog, for example if you already have an extension with the same name:
+
+.Changing the default extension name
+====
+include::sample[dir="snippets/dependencyManagement/catalogs-toml/groovy",files="settings.gradle[tags=change_default_extension_name]"]
+include::sample[dir="snippets/dependencyManagement/catalogs-toml/kotlin",files="settings.gradle.kts[tags=change_default_extension_name]"]
```

Comment:
Something's broken with the tags here

Reply:
Weird I can see the sample...

Reply:
Mh. Now it works. Maybe it was caused by some other formatting problem further up.